spec/support/generators.rb

```diff
@@ -0,0 +1,22 @@
+module Pageflow
+  module Specs
+    module Generators
+      module Macros
+        def setup_default_destination
+          # Tell the generator where to put its output (what it thinks of as Rails.root)
+          destination File.expand_path('../../tmp', __FILE__)
+
+          before { prepare_destination }
+        end
+      end
+
+      def self.included(klass)
+        klass.extend(Macros)
+      end
```

Comment:
What do you think of turning this into a shared context with metadata `type: generator`. That way we get rid of alle the `setup_default_destination` calls.